src/_includes/nav.html

```diff
@@ -1,40 +1,57 @@
-                    <button type="button" class="btn-close p-0 m-0 rounded-0 border-0 opacity-100" data-bs-dismiss="modal" aria-label="Close"></button>
-                </div>
-                <div class="modal-body">
-                    <div class="d-flex align-items-center h-100 offset-lg-2 col-lg-10">
-                        <ul class="list-unstyled">
-                            <li>
-                                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/">Home</a>
-                            </li>
-                            <li>
-                                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/about">About</a>
-                            </li>
-                            <li>
-                                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/jobs">Jobs</a>
-                            </li>
-                            <li>
-                                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/blog">Blog</a>
-                            </li>
-                            <li>
-                                <a class="h2 text-white d-inline-block" target="_blank" href="/capabilities">Capabilities Statement</a>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-                <div class="modal-footer border-0 p-0">
-                    <div class="nav-footer-pattern"></div>
-                </div>
-            </div>
+  <a class="d-inline-block" href="/"
+    ><img width="88" height="46" src="/assets/compiler_logo_stacked.svg" alt="Compiler logo"
+  /></a>
+  <button
+    class="navbar-toggler p-0 border-0 rounded-0"
+    type="button"
+    data-bs-toggle="modal"
+    data-bs-target="#navbarToggler"
+    aria-controls="navbarToggler"
+    aria-expanded="false"
+    aria-label="Toggle navigation"
+  >
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="modal fade" tabindex="-1" id="navbarToggler" aria-label="menu">
+    <div class="modal-dialog modal-fullscreen">
+      <div class="modal-content">
+        <div class="modal-header border-0 d-flex justify-content-end">
+          <button
+            type="button"
+            class="btn-close p-0 m-0 rounded-0 border-0 opacity-100"
+            data-bs-dismiss="modal"
+            aria-label="Close"
+          ></button>
         </div>
+        <div class="modal-body">
+          <div class="d-flex align-items-center h-100 offset-lg-2 col-lg-10">
+            <ul class="list-unstyled">
+              <li>
+                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/">Home</a>
+              </li>
+              <li>
+                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/about">About</a>
+              </li>
+              <li>
+                <a class="h2 text-white d-inline-block mb-4 pb-2 pb-lg-0" href="/our-work">Our Work</a>
```

Comment:
Just double-checking, is Our Work title-cased or sentence-cased? Don't see it in Figma.

Reply:
Thanks for checking @machikoyasuda, yep, we noticed that the mockup in Figma hadn't been updated, it doesn't have `Blog` or `Our Work` on it ([Issue 215](https://github.com/compilerla/compiler.la/issues/215)). So we decided to add `Our Work` after `About` and use title-case to match the `Capabilities Statement` menu item.